ci/delete_thin-edge/delete_thin_edge.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
```

Comment:
The script name must be consistent with `get-thin-edge_io.sh`.

My proposal is `delete-thin-edge_io.sh`.

Reply:
make sense to keep them consistent. Fixed.